src/app/pages/home/about/about.component.html

```diff
@@ -1 +1,41 @@
-<div>About Page</div>
\ No newline at end of file
+<div class="aboutus-overlay1">
+    <div class="aboutus-overlay1-content">
+      <div class="headline">About NTU Open Source Society</div>
+      <p>NTU Open Source Society (OSS) is a student community at the Nanyang Technological University helping to promote the value and importance of Open Source in NTU. We attract students of common technical interests to foster a community of engineers, hackers and builders to explore their interests in technology and share their knowledge and expertise with others. Through our weekly meetings, we help students learn practical coding skills, find partners to work with on interesting projects and also help promote the value and importance of contributing to open source development. Regardless of skill level or expertise, if you like to build things, this is the place for you.</p>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
```

Comment:
The correct class to specify a column is `col`. This automatically positions itself as if it's a table when placed inside a `row`. Like the border styling for our `row`, use the following styles to define a border to the first `col` element.

```scss
.col {
    padding: 2rem;
    &:first-of-type {
        border-right: 1px solid #BABABA;
    }
}
```